### PR TITLE
Adds Git to the modules

### PR DESCRIPTION
**Pull request by orowith2os**
Should close https://github.com/flathub/org.polymc.PolyMC/issues/21

### CODE REVIEW

**Comment by orowith2os** [comment]
bot, build org.polymc.PolyMC

**Comment by orowith2os** [comment]
Well, that was an experience.

**Comment by orowith2os** [comment]
Flathub seems to be having a few issues right now, gonna give this a while.

**Comment by orowith2os** [comment]
bot, build org.polymc.PolyMC

**Comment by orowith2os** [comment]
bot, build org.polymc.PolyMC

**Comment by orowith2os** [comment]
> flatpak run --env=G_DEBUG=fatal-criticals org.freedesktop.appstream-glib validate builddir/*/share/appdata/org.polymc.PolyMC.appdata.xml
 in dir /srv/buildbot/worker/build-x86_64-5/build (timeout 1200 secs)
 watching logfiles {}
 argv: b'flatpak run --env=G_DEBUG=fatal-criticals org.freedesktop.appstream-glib validate builddir/*/share/appdata/org.polymc.PolyMC.appdata.xml'
 using PTY: False
builddir/files/share/appdata/org.polymc.PolyMC.appdata.xml: FAILED:
• attribute-invalid     : <screenshot> width (931) did not match specified (976) [https://polymc.org/img/screenshots/LauncherDark.png]
• attribute-invalid     : <screenshot> height (759) did not match specified (764) [https://polymc.org/img/screenshots/LauncherDark.png]
• attribute-invalid     : <screenshot> width (777) did not match specified (1083) [https://polymc.org/img/screenshots/PropertiesDark.png]
• attribute-invalid     : <screenshot> height (693) did not match specified (735) [https://polymc.org/img/screenshots/PropertiesDark.png]
• attribute-invalid     : <screenshot> width (1018) did not match specified (1000) [https://polymc.org/img/screenshots/ModInstallDark.png]
• attribute-invalid     : <screenshot> height (858) did not match specified (692) [https://polymc.org/img/screenshots/ModInstallDark.png]
• attribute-invalid     : <screenshot> width (860) did not match specified (1103) [https://polymc.org/img/screenshots/ModpackInstallDark.png]
• attribute-invalid     : <screenshot> height (848) did not match specified (954) [https://polymc.org/img/screenshots/ModpackInstallDark.png]
Validation of files failed
program finished with exit code 1
elapsedTime=2.117558

This is where it failed, seems to be an issue unrelated to this PR.

**Comment by orowith2os** [comment]
@dada513 care to take a look?

File: org.polymc.PolyMC.yml [hunk 1]
```diff
@@ -28,6 +28,17 @@ finish-args:
...
 modules:
+  # Git, closes https://github.com/flathub/org.polymc.PolyMC/issues/21
+  - name: git
+    buildsystem: simple
+    build-commands:
+      - make prefix=/app install
+    sources:
+      - type: git
+        url: https://github.com/git/git.git
+        commit: ac8035a2affdf30f2c691ad760826d955bba0507
```
Comment (nanonyme) [inline org.polymc.PolyMC.yml:39]: 1. Use tags
2. Use flatpak external data checker for automatic updates

Git frequently has CVE's that require immediate reaction. It must be trivially knowable if the app contains vulnerable version.

Comment (nanonyme) [reply org.polymc.PolyMC.yml:39]: See https://github.com/flathub/flatpak-external-data-checker for 2

**Comment by orowith2os** [comment]
As this flatpak has been taken over by developers that I would rather not interact with, and it's dead anyways, I'll close this MR and instead make it to the PrismLauncher flatpak should it need it.

**Pull request closed without merging**